### PR TITLE
try except cancel

### PR DESCRIPTION
**Pull request by rootedbox**

### CODE REVIEW

File: limit-orders.js [hunk 1]
```diff
@@ -335,7 +335,12 @@ async function getNewBlocks(fromBlock) {
 }
 
 async function getMinAmount(depositor, deposit_id) {
-    let amount = await contractInstance.methods.cancel(deposit_id, 0).call({from: depositor});
+    let amount = 0;
+    try {
+        let amount = await contractInstance.methods.cancel(deposit_id, 0).call({from: depositor});
```
Comment (webelf101) [inline limit-orders.js:340]: We should remove `let` here, shouldn't we?

**Review by webelf101 — CHANGES_REQUESTED** [review]
We should remove `let` in Line 340, shouldn't we?

**Comment by rootedbox** [comment]
fixed

**Review by webelf101 — APPROVED** [review]
LGTM

**Pull request merged**